### ISSUE TITLE
implementation details for ACDC

### ISSUE THREAD

**Issue by kathyliu579** [HuCaoFighting/Swin-Unet#56]
i use the same paramater like SGD, lr =0.05 for ACDC. but the loss can not be reduced when loss = 0.2
Please tell me what are your training parameters?

**Comment by XL-H** [comment]
> i use the same paramater like SGD, lr =0.05 for ACDC. but the loss can not be reduced when loss = 0.2
> Please tell me what are your training parameters?

  @kathyliu579 Hi, would you share the preprocessed ACDC dataset, we get works on it, 😊

**Comment by XL-H** [comment]
> i use the same paramater like SGD, lr =0.05 for ACDC. but the loss can not be reduced when loss = 0.2
> Please tell me what are your training parameters?

I'm facing the same problem  as you ever met, Why this happened?

**Comment by gail-yxie** [comment]
> 

Hi, would you please share the preprocessed ACDC data with me? My email is yuege@oden.utexas.edu.  Many Thanks!

**Comment by XL-H** [comment]
> Hi, would you please share the preprocessed ACDC data with me? My email is [yuege@oden.utexas.edu](mailto:yuege@oden.utexas.edu). Many Thanks!

3579628328，你加我qq吧，我发一份我改的预处理的代码给你，预处理的ACDC等不到回复的...

**Comment by XL-H** [comment]
[ACDC-Preprocess](https://github.com/XL-H/ACDC_preprocess)

**Comment by wangqingbin** [comment]
> Hi, would you please share the preprocessed ACDC data with me? My email is [yuege@oden.utexas.edu](mailto:yuege@oden.utexas.edu). Many Thanks!

Hi, would you please share the preprocessed ACDC data with me? My email is qbwang@whu.edu.cn. Many Thanks!

**Comment by XL-H** [comment]
Hi, check this link&nbsp;https://github.com/XL-H/ACDC_preprocess




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "HuCaoFighting/Swin-Unet"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年10月12日(星期三) 下午4:24
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [HuCaoFighting/Swin-Unet] implementation details for ACDC (Issue #56)





  
Hi, would you please share the preprocessed ACDC data with me? My email is ***@***.*** Many Thanks!
  
Hi, would you please share the preprocessed ACDC data with me? My email is ***@***.*** Many Thanks!

**Comment by wangqingbin** [comment]
> Hi, check this link&nbsp;https://github.com/XL-H/ACDC_preprocess
> […](#)
> ------------------&nbsp;原始邮件&nbsp;------------------ 发件人: "HuCaoFighting/Swin-Unet" ***@***.***&gt;; 发送时间:&nbsp;2022年10月12日(星期三) 下午4:24 ***@***.***&gt;; ***@***.******@***.***&gt;; 主题:&nbsp;Re: [HuCaoFighting/Swin-Unet] implementation details for ACDC (Issue #56) Hi, would you please share the preprocessed ACDC data with me? My email is ***@***.*** Many Thanks! Hi, would you please share the preprocessed ACDC data with me? My email is ***@***.*** Many Thanks! — Reply to this email directly, view it on GitHub, or unsubscribe. You are receiving this because you commented.Message ID: ***@***.***&gt;

Thanks, but where can I download the ACDC dataset? Or could you the ACDC data to me? My email is [qbwang@whu.edu.cn](mailto:qbwang@whu.edu.cn). Many Thanks!

**Comment by wangqingbin** [comment]
> Hi, check this link&nbsp;https://github.com/XL-H/ACDC_preprocess
> […](#)
> ------------------&nbsp;原始邮件&nbsp;------------------ 发件人: "HuCaoFighting/Swin-Unet" ***@***.***&gt;; 发送时间:&nbsp;2022年10月12日(星期三) 下午4:24 ***@***.***&gt;; ***@***.******@***.***&gt;; 主题:&nbsp;Re: [HuCaoFighting/Swin-Unet] implementation details for ACDC (Issue #56) Hi, would you please share the preprocessed ACDC data with me? My email is ***@***.*** Many Thanks! Hi, would you please share the preprocessed ACDC data with me? My email is ***@***.*** Many Thanks! — Reply to this email directly, view it on GitHub, or unsubscribe. You are receiving this because you commented.Message ID: ***@***.***&gt;

Thanks, but where can I download the ACDC dataset? Or could you send the ACDC data to me? My email is [qbwang@whu.edu.cn](mailto:qbwang@whu.edu.cn). Many Thanks!

**Comment by XL-H** [comment]
Sorry that I can't share it to you directly, but you can download it here,&nbsp;https://acdc.creatis.insa-lyon.fr/description/databases.html




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "HuCaoFighting/Swin-Unet"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年10月12日(星期三) 下午4:55
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [HuCaoFighting/Swin-Unet] implementation details for ACDC (Issue #56)





  
Hi, check this link&nbsp;https://github.com/XL-H/ACDC_preprocess
 …
 ------------------&nbsp;原始邮件&nbsp;------------------ 发件人: "HuCaoFighting/Swin-Unet" @.&gt;; 发送时间:&nbsp;2022年10月12日(星期三) 下午4:24 @.&gt;; @.@.&gt;; 主题:&nbsp;Re: [HuCaoFighting/Swin-Unet] implementation details for ACDC (Issue #56) Hi, would you please share the preprocessed ACDC data with me? My email is @.*** Many Thanks! Hi, would you please share the preprocessed ACDC data with me? My email is @.*** Many Thanks! — Reply to this email directly, view it on GitHub, or unsubscribe. You are receiving this because you commented.Message ID: @.***&gt;
  
Thanks, but where can I download the ACDC dataset? Or could you the ACDC data to me? My email is ***@***.*** Many Thanks!

**Comment by wangqingbin** [comment]
> Sorry that I can't share it to you directly, but you can download it here,&nbsp;https://acdc.creatis.insa-lyon.fr/description/databases.html
> […](#)
> ------------------&nbsp;原始邮件&nbsp;------------------ 发件人: "HuCaoFighting/Swin-Unet" ***@***.***&gt;; 发送时间:&nbsp;2022年10月12日(星期三) 下午4:55 ***@***.***&gt;; ***@***.******@***.***&gt;; 主题:&nbsp;Re: [HuCaoFighting/Swin-Unet] implementation details for ACDC (Issue #56) Hi, check this link&nbsp;https://github.com/XL-H/ACDC_preprocess … ------------------&nbsp;原始邮件&nbsp;------------------ 发件人: "HuCaoFighting/Swin-Unet" @.&gt;; 发送时间:&nbsp;2022年10月12日(星期三) 下午4:24 @.&gt;; @.@.&gt;; 主题:&nbsp;Re: [HuCaoFighting/Swin-Unet] implementation details for ACDC (Issue #56) Hi, would you please share the preprocessed ACDC data with me? My email is @.*** Many Thanks! Hi, would you please share the preprocessed ACDC data with me? My email is @.*** Many Thanks! — Reply to this email directly, view it on GitHub, or unsubscribe. You are receiving this because you commented.Message ID: @.***&gt; Thanks, but where can I download the ACDC dataset? Or could you the ACDC data to me? My email is ***@***.*** Many Thanks! — Reply to this email directly, view it on GitHub, or unsubscribe. You are receiving this because you commented.Message ID: ***@***.***&gt;

Thank you very much!
